controllers/loadtest_controller.go

```diff
@@ -125,6 +125,70 @@ func (r *LoadTestReconciler) Reconcile(req ctrl.Request) (ctrl.Result, error) {
 	return ctrl.Result{}, nil
 }
 
+// CheckMissingPods attempts to check if any pods is missing for current
+// LoadTest.
+func CheckMissingPods(currentLoadTest *grpcv1.LoadTest, allRunningPods *corev1.PodList) []*grpcv1.Component {
+
+	//Creat maps for Clients and Servers for current currentLoadTest
```

Comment:
Creat -> Create. "for current currentLoadTest" is probably not what you want to say.

Reply:
I will definitely check all my spelling more carefully!